README.md

```diff
@@ -35,6 +36,11 @@ pdm run docker
 ## Setup
 
 ```
+# Install YAJL library
```

Comment:
Creating a new markdown section could be even better, in my opinion. Like `### Install Yajl``